mpc_follower_wrapper/src/mpc_follower_wrapper.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2018-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include "mpc_follower_wrapper/mpc_follower_wrapper.hpp"
+
+namespace mpc_follower_wrapper {
+
+MPCFollowerWrapper::MPCFollowerWrapper(ros::NodeHandle &nodeHandle): nh_(nodeHandle)
+{
+  if (!ReadParameters())
+  {
+    ROS_ERROR("Could not read parameters.");
+  }
+
+  Initialize();
+
+  ROS_INFO("Successfully launched node.");
+}
+
+MPCFollowerWrapper::~MPCFollowerWrapper() {
+}
+
+void MPCFollowerWrapper::Initialize() {
+  // SystemAlert Subscriber
+  system_alert_sub_ = nh_.subscribe("system_alert", 10, &MPCFollowerWrapper::SystemAlertHandler, this);
+  // SystemAlert Publisher
+  system_alert_pub_ = nh_.advertise<cav_msgs::SystemAlert>("system_alert", 10, true);
+
+  // Pose Subscriber
+  pose_sub.subscribe(nh_, "current_pose", 1);
+  // Trajectory Plan Subscriber
+  trajectory_plan_sub.subscribe(nh_, "trajectory_plan", 1);
+
+  // WayPoints Publisher
+  way_points_pub_ = nh_.advertise<autoware_msgs::Lane>("final_waypoints", 10, true);
+}
+
+bool MPCFollowerWrapper::ReadParameters() {
```

Comment:
I don't see the point of this. It always returns true and the one time it is used is before Initialize is called so if it did what the comment said it really should have returned false

Reply:
I wrote this node based on pure pursuit wrapper and this was part of that, so how fo you suggest I modify it?

Reply:
Seems you could just get rid of this function entirely